plugins/modules/oneview_id_pools_ipv4_range_facts.py

```diff
@@ -149,7 +150,7 @@ def __init__(self):
             name=dict(required=False, type='str'),
             uri=dict(required=False, type='str'),
             subnetUri=dict(required=False, type='str'),
-            options=dict(required=False, type='list'),
+            options=dict(required=False, type='list', elements='str'),
```

Comment:
why only one place? should not we do it for others too?

Reply:
Yes Avijit. I have already made changes in all places applicable.